### ISSUE TITLE
How to specify forward merge strategy? 

### ISSUE THREAD

**Issue by JobaDiniz** [pascalgn/automerge-action#183]
Is there a way to specify merge with `--ff-only`?

**Comment by pascalgn** [comment]
No, this is currently not possible. Option `MERGE_METHOD: "rebase"` will merge without merge commit, but it does not check if HEAD is up-to-date first (what ff-only does)

Feel free to create a PR for this!
